packages/catalog-process/src/utilities/errorMappers.ts

```diff
@@ -80,6 +82,7 @@ export const deleteDraftDescriptorErrorMapper = (
       "eServiceDescriptorNotFound",
       () => HTTP_STATUS_NOT_FOUND
     )
+    .with("notValidDescriptor", () => HTTP_STATUS_BAD_REQUEST)
```

Comment:
This error has been added to handle the `notValidDescriptor` error that is now raised when trying to delete a descriptor that is not in `draft` state.